src/redux/actions/profile/fetchProfile.js

```diff
@@ -0,0 +1,52 @@
+import { toast } from 'react-toastify';
+import * as profileActionCreators from './profileAction';
+import { BASE_URL, VERSION } from '../actionTypes';
+
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwiZW1haWwiOiJ1c2VyZm91ckBnbWFpbC5jb20iLCJ1c2VybmFtZSI6InVzZXJmb3VyIiwicm9sZSI6Im5vcm1hbCIsInZlcmlmaWVkIjp0cnVlLCJpYXQiOjE1NzA2OTkyNjQsImV4cCI6MTU3MDc4NTY2NH0.2b1Qtkjd7PCaeV9AiydE5xP7YVAgP0xAAVSj14qS3nE';
```

Comment:
You can now make the token dynamic. I think it might be better put it as an argument in fetchProfile function

Reply:
@SokoPaulSokool I have made the token dynamic as well as making the route protected.